deploy/lts-vuln-monitor/report.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+# Copyright 2021 The Skaffold Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script creates a github issue if it hasn't been created when there
+# are vulnerabilities found in the LTS image.
+
+set -x
+
+# Variables that will be substituted in cloudbuild.yaml.
```

Comment:
Note that these substitutions must be [explicitly propagated into the environment](https://cloud.google.com/build/docs/configuring-builds/substitute-variable-values#using_default_substitutions) with an `env:` section on a step in the cloudbuild.yaml

Reply:
ah, I only tested with default values before. Added the substitutions section.